_posts/2020-06-27-lsp-explained.md

```diff
@@ -0,0 +1,183 @@
+---
+title: The Liskov Substitution Principle Explained
+categories: [craft]
+date: 2020-06-27 05:00:00 +1100
+modified: 2020-06-27 05:00:00 +1100
+author: saajan
```

Comment:
Please add yourself to `authors.yml` (preferably with a picture), otherwise you won't show up as an author under the article.

Reply:
Done, thanks. :)